### PR TITLE
Feature/ml #34 layout

### LINKED ISSUE #34: Create a Layout Component with Footer
- [x] Write Footer

- [x] Create Layout Component 

- [x] Add Header inside Layout Component

### PR DESCRIPTION
**Pull request by romanazaruk**

### CODE REVIEW

File: frontend/src/components/pages/settings/Settings.tsx [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import React from 'react';
+import Layout from '../../shared/layout/Layout';
 
 const Settings: React.FC = () => {
-  return <h1>Settings</h1>;
+  return (
+    <Layout PageTitle="Settings">
```
Comment (apastuhov) [inline frontend/src/components/pages/settings/Settings.tsx]: lowercase

File: frontend/src/components/shared/footer/Footer.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@import '../../../styles/variables.scss';
+
+footer {
+  .footer-text {
```
Comment (apastuhov) [inline frontend/src/components/shared/footer/Footer.scss]: just use `p`

File: frontend/src/components/shared/footer/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    width: 200px;
+    text-align: center;
+    margin: auto;
```
Comment (apastuhov) [inline frontend/src/components/shared/footer/Footer.scss]: margin: 15px auto 0px;

File: frontend/src/components/shared/footer/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Footer.scss';
+
+const Footer: React.FC = () => {
+  return (
+    <footer>
+      <p className="footer-text"> © 2020 Sigma Software Privacy Policy Terms of use</p>
```
Comment (apastuhov) [inline frontend/src/components/shared/footer/Footer.tsx]: remove `p` and add `br`

File: frontend/src/components/shared/layout/Layout.scss [hunk 5]
```diff
@@ -0,0 +1,12 @@
+@import '../../../styles/variables.scss';
+
+.container {
+  max-width: 944px;
```
Comment (apastuhov) [inline frontend/src/components/shared/layout/Layout.scss]: variable

File: frontend/src/components/shared/layout/Layout.scss [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+  margin: 26px auto;
+  .page-title {
+    font-weight: 500;
+    font-size: 22px;
```
Comment (apastuhov) [inline frontend/src/components/shared/layout/Layout.scss]: use mixing

File: frontend/src/components/shared/layout/Layout.tsx [hunk 7]
```diff
@@ -0,0 +1,24 @@
+import React, { ReactNode } from 'react';
+import Header from '../header/Header';
+import Footer from '../footer/Footer';
+import './Layout.scss';
+
+type Props = {
```
Comment (apastuhov) [inline frontend/src/components/shared/layout/Layout.tsx]: google it:
interface IProps extends React.Props {
  pageTitle: string;
}

**Review by apastuhov — CHANGES_REQUESTED** [review]
LGTM

**Review by OrestHaponov — APPROVED** [review]
good!

**Review by grok-rs — APPROVED** [review]

**Pull request merged**